### PR TITLE
Zar 11 add category entity

### PR DESCRIPTION
**Pull request by Sidanton6006**

### CODE REVIEW

File: Core/Entities/CategoryEntity.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+public class CategoryEntity : BaseEntity
+{
+    public CategoryEntity()
+    {
+        Children = new HashSet<CategoryEntity>();
+        Products = new HashSet<ProductEntity>();
+    }
+    public string Title { get; set; }
+    public string Image { get; set; }
+    public Guid? ParentId { get; set; }
```
Comment (Mike-Opanasiuk) [inline Core/Entities/CategoryEntity.cs]: Rename to ParentCategoryId

**Review by byCodeXp — CHANGES_REQUESTED** [review]
📛 Use Guid instead of int type for CategoryId in ProductEntity.cs

**Review by Mike-Opanasiuk — CHANGES_REQUESTED** [review]
Request changes

**Review by Mike-Opanasiuk — APPROVED** [review]
Well done!

**Review by byCodeXp — APPROVED** [review]
Ok

**Pull request merged**